how-to/synonyms.md

```diff
@@ -0,0 +1,151 @@
+---
+title: How to synonyms search
+---
+
+## How to synonym search
+
+We need to make a new table for register synonyms to search synonym.
+
+We explain how to register synonyms, delete synonyms, update synonyms, search synonyms, and make a table for register synonyms.
+
+### How to make a new table to register synonyms
+
+We make a table to register synonyms as below.
+
+```sql
+CREATE TABLE synonyms (
+  term text PRIMARY KEY,
+  synonyms text[]
+);
+
+CREATE INDEX synonyms_search ON synonyms USING pgroonga (term pgroonga.text_term_search_ops_v2);
+```
+
+We can get a term that registered to `synonyms` by using a term that resitered to `term` as key.
+
```

Comment:
(Hmm,  It may be hard to understand what this sentense mean, but I have no idea how to rewrite...)

Reply:
We can get a term that is registered in `synonyms` with a key that is registered in `term`. ?